code/game/machinery/portable_turret.dm

```diff
@@ -51,13 +44,12 @@
 
 	var/attacked = 0		// if set to 1, the turret gets pissed off and shoots at people nearby (unless they have sec access!)
 
-	//var/emagged = 0			// 1: emagged, 0: not emagged
 	var/on = 1				// determines if the turret is on
 	var/disabled = 0
 
 	var/datum/effect/effect/system/spark_spread/spark_system // the spark system, used for generating... sparks?
 
-	machine_flags = EMAGGABLE
+	machine_flags = EMAGGABLE | SHUTTLEWRENCH
```

Comment:
They already were, I only added the flag so it would keep the same functionality when changed to using wrenchAnchor() checks